### ISSUE TITLE
trimTrailingZeros and negative numbers

### ISSUE THREAD

**Issue by danilort** [nortakales/vs-code-qalc#12]
Qalc 0.1.8     
(I love this extension)

Using these settings:

```
"qalc.output.trimTrailingZeros": true
"qalc.output.precision": 4
```
The negative numbers ignore the option trimTrailingZeros:

![immagine](https://user-images.githubusercontent.com/16148184/147380864-74c789b3-95b7-4f9d-8b2d-8df21d262c66.png)

Thank's for your job!

**Comment by danilort** [comment]
I suggest to modify the file out/formatter.js:

```
if (formatterSettings.trimTrailingZeros) {
    // Make sure there is a decimal value
    if (/^\d+\.\d+$/.test(output)) {
        output = output.replace(/0+$/, '');
    }
    .............
```

to allow the "-" character:

```
if (/^-?\d+\.\d+$/.test(output))  {
```

**Comment by nortakales** [comment]
Fixed in Qalc 0.1.9
